g2p_registry_rest_api/models/group_membership.py

```diff
@@ -18,19 +16,7 @@ class GroupMembersInfoOut(NaiveOrmModel):
     ] = None  # TODO: Would be nicer to have it as a list of str
 
 
-class GroupMembersInfoIn(NaiveOrmModel):
-    name: str
-    given_name: str = None
-    family_name: str = None
-    ids: List[RegistrantIDIn] = None
-    registration_date: date = None
-    phone_numbers: List[PhoneNumberIn] = None
-    email: str = None
-    address: str = None
-    gender: str = None
-    birthdate: date = None
-    birth_place: str = None
-    is_group = False
```

Comment:
Any reason for removing this ?

Reply:
This is just removing fields that are already present in `IndividualInfoIn` model, and adding that model to this as parent.
```
class GroupMembersInfoIn(IndividualInfoIn):
```
[Next Line](https://github.com/OpenG2P/openg2p-registry/blob/e29ac4e1c666ea7c03da93de217bbbc1e3803ddc/g2p_registry_rest_api/models/group_membership.py#L19)

Reply:
I do not see it added.
But by doing this, we are breaking the other apps that depend on it. So we should have a good reason to do this, and make sure we do not break the other apps.